contracts/scripts/lifebank.sh

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env bash
+# Local .env
+if [ -f ../../.env ]; then
+    # Load Environment Variables
+    export $(cat ../../.env | grep -v '#' | awk '/=/ {print $1}')
+fi
+set -e
+unlock_lifebank_wallet() {
+    echo 'Unlocking Wallet...'
+    cleos wallet unlock -n lifebank --password $(cat ./secrets/lifebank_wallet_password.txt) ||
+        echo "Wallet has already been unlocked..."
+}
+create_lifebank_wallet() {
+    echo 'Creating Wallet...'
+    cleos wallet create -n lifebank --to-console |
+        awk 'FNR > 3 { print $1 }' |
+        tr -d '"' \
+            >./secrets/lifebank_wallet_password.txt
+    cleos wallet open -n lifebank
+    unlock_lifebank_wallet
+    cleos wallet import -n lifebank --private-key $EOS_PRIV_KEY
+}
+create_lifebank_accounts() {
+    echo 'Creating Lifebank Accounts...'
+    lifebank_accounts=(
+        "consent2life"
+        "lifebankcode"
+        "lifebankcoin"
+        "lbacccreator"
+        "sponsprueba1"
+        "donorprueba1"
+        "12letterlife"
+    )
+    for account in "${lifebank_accounts[@]}"; do
+        echo "Creating lifebank account '$account'"
+        keys=($(cleos create key --to-console))
+        pub=${keys[5]}
+        priv=${keys[2]}
+        cleos wallet import -n lifebank --private-key $priv
+        mkdir -p ./secrets
+        echo $pub >./secrets/$account.pub
+        echo $priv >./secrets/$account.priv
+        cleos -u $EOS_API_URL system newaccount eosio \
+            --transfer $account \
+            $EOS_PUB_KEY \
+            --stake-net "10000 EOS" \
+            --stake-cpu "10000 EOS" \
+            --buy-ram-kbytes 8192
+    done
+}
+assign_resources() {
+    cleos -u $EOS_API_URL get account lbacccreator 
+    cleos -u $EOS_API_URL system buyram eosio lbacccreator "10000 EOS"
+    cleos -u $EOS_API_URL get account lbacccreator 
+    cleos -u $EOS_API_URL transfer eosio lbacccreator "10000000 EOS" "Lifebank initial funding"
+}
+deploy_lifebank_contracts() {
+    echo 'Deploy Consent Contract'
+    cleos -u $EOS_API_URL set contract consent2life ../consent2life/ -p consent2life@active -j -s -d
```

Comment:
we need to add the writer run action  and sign the transaction 

`-s` = skip signatures